tests/page-objects/send-form.selectors.ts

```diff
@@ -7,12 +7,32 @@ export enum SendFormSelectors {
   InputRecipientField = 'input-recipient-field',
   InputRecipientFieldErrorLabel = 'input-recipient-field-error-label',
 
-  FeeEstimateItem = 'fee-estimate-item',
+  MemoField = 'memo-field',
+  MemoFieldErrorLabel = 'memo-field-error-label',
+
+  FeeEstimateItem = 'Standard-Fees',
   FeeEstimateSelect = 'fee-estimate-select',
+  LowFeesOption = 'Low-Fees',
+  HighFeesOption = 'High-Fees',
+  StandardFeesOption = 'Standard-Fees',
+  CustomFeesOption = 'Custom-Fees',
```

Comment:
I think all these should be singular? `LowFeeOption`, etc. ...`Standard-Fee`, `Low-Fee`, etc.

Reply:
Also, some of these keys duplicate others in the list, "Standard-Fees" exists twice